.github/workflows/ci.yml

```
@@ -14,6 +14,17 @@ env:
   IMAGE_NAME: maykinmedia/open-inwoner
 
 jobs:
+  i18n_and_bundle:
```

This name doesn't explain what this part does.

```suggestion
  test_frontend:
```